packages/postgres-database/tests/conftest.py

```diff
@@ -268,10 +267,11 @@ def create_fake_user(sync_engine: sqlalchemy.engine.Engine) -> Iterator[Callable
     async def _creator(
         conn: SAConnection, group: RowProxy | None = None, **overrides
     ) -> RowProxy:
-        user_id = await conn.scalar(
-            users.insert().values(**random_user(**overrides)).returning(users.c.id)
+
```

Comment:
we do have pytest-simcore `create_registered_user` that does the exact same thing